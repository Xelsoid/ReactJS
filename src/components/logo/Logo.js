import React from 'react';

import './logo.scss';

const Logo = () => (
  <h1 className = "logo">
    <a className = "logo__link" href = "http:/google.com">Movie Finder</a>
  </h1>
);

export default Logo;
