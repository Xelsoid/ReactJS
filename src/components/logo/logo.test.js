import React from 'react';
import Logo from './Logo';

it('renders correctly', () => {
  const component = shallow(
    <Logo />
  );
  expect(component).toMatchSnapshot();
});

