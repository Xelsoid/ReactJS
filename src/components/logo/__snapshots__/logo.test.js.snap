// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<h1
  className="logo"
>
  <a
    className="logo__link"
    href="http:/google.com"
  >
    Movie Finder
  </a>
</h1>
`;
